src/toxcore/tcp/server/server_ext.rs

```diff
@@ -31,24 +32,32 @@ impl ServerExt for Server {
     fn run(self: Self, listner: TcpListener, dht_sk: SecretKey) -> IoFuture<()> {
```

Comment:
Add notice that ServerExt::run should run inside `tokio::run`

Reply:
done